Servers/swagger.yaml

```
@@ -3869,70 +3984,126 @@ paths:
                   type: object
   /users/login:
     post:
+      summary: Authenticate user and receive JWT tokens
+      description: |
+        Authenticates a user with email and password credentials. Returns both access and refresh tokens.
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)